docs/testing.rst

```diff
@@ -51,5 +51,35 @@ performance by running the following command.
 .. sourcecode:: sh
 
    python tests/benchmarks.py
+   
+Makefile Utility
+----------------
+
+Makefiles are a simple way to perform code compilation on ``Linux platforms``.
+
+We often forget to compile or to run all tests whenever we make any change in existing files,
```

Comment:
I think it can write like this

``We often forgot to build docs, run nosetes or Django tests``